test/spec/directives/focus-me.directive.spec.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+describe('focusMe Directive', function() {
+  var element, scope;
+
+  beforeEach(module('habitrpg'));
+
+  beforeEach(inject(function($rootScope, $compile) {
+    scope = $rootScope.$new();
+
+    element = "<input focus-me></input>";
+
+    element = $compile(element)(scope);
+    scope.$digest();
+  }));
+
+  it('focuses the element when appended to the DOM', function(done) {
+    inject(function($timeout) {
+      element.appendTo(document.body);
+
+      element.on('focus', function() { done() });
+
+      $timeout.flush();
```

Comment:
This is all very concise, but I think it'd be more human-readable to do something like this:

``` javascript
var focusSpy = sinon.spy();

element.appendTo(document.body);
element.on('focus', focusSpy);
$timeout.flush();

expect(focusSpy).to.be.called;
done();
```

Thoughts?